### ISSUE TITLE
Allow Join with CopyValue in tasks file

### ISSUE THREAD

**Issue by eduardomourar** [org-formation/org-formation-cli#79]
Take the following snippet that exports a value using the default property of a resource using the `!Ref` intrinsic function:

```yaml
AWSTemplateFormatVersion: 2010-09-09

Resources:

  ElasticIpNatGateway:
    Type: AWS::EC2::EIP
    Properties:
      Domain: vpc

Outputs:

  NatGatewayIp:
    Description: The NAT Gateway public IP
    Value: !Ref ElasticIpNatGateway
    Export:
      Name: 'exported-ngw-ip'
```

Whenever trying to use `!Join [ '', [ !CopyValue 'exported-ngw-ip', '/32' ] ]`, the following is being passed as string to CloudFormation: `[object Object]`

We can use the `!ImportValue` as a workaround if the stack is located in the same account and region.

**Comment by OlafConijn** [comment]
:-) problem was not with `!CopyValue`... problem was that `!Join` is/was not supported.
Workaround could have been to use `!Sub` instead of `!Join`. anyhow, just committed  support for `!Join`. 

Published as `aws-organization-formation@0.9.7-beta.5` just now.
commit here: https://github.com/OlafConijn/AwsOrganizationFormation/commit/29b88ff33910f9157cd7b0514d1cb48431226595

**Comment by OlafConijn** [comment]
fixed in v0.9.7
